ChangeLog.md

```diff
@@ -5,6 +5,8 @@
 
   * Add student name and clickable e-mail address information to issue reports (James Balamuta).
 
+  * Add `Makefile` to main directory to handle docker builds and previews (James Balamuta).
```

Comment:
I like this Makefile, but I'm slightly nervous about the fact that it won't work out of the box on Windows. Is this somewhere where we should instead be using more npm scripts? See https://www.keithcirkel.co.uk/how-to-use-npm-as-a-build-tool/

Reply:
I'm very much on board with working towards eliminating our Makefile - npm scripts were designed for these kind of things.

Reply:
Unfortunately I just read more about npm scripts. They kind of suck compared to Makefiles -- why can't people at least retain the same functionality when reinventing the wheel?!? :-)

But I'm still generally in favor of them wherever possible.

Reply:
I'll have to look a bit more into the process. But, I agree with this sentiment since interoperability between macOS, Windows, and Linux distros are paramount.